app/ui/browser/model/index.js

```diff
@@ -25,6 +25,21 @@ export const Pages = Immutable.Record({
 }, 'Pages');
 
 /**
+ * The metadata parsed from a page for a Page record.
+ */
+export const PageMeta = Immutable.Record({
+  description: undefined,
+  icon_url: undefined,
+  image_url: undefined,
+  title: undefined,
+  type: undefined,
+  url: undefined,
+  player_url: undefined,
+  player_stream_url: undefined,
+  player_stream_content_type: undefined,
```

Comment:
I think these should be part of some sort of a "super record", not directly part of a PageMeta, which I see more of a base class for such metadata. Everything is fine except the `player_*` rules.